packages/clerk-js/src/core/clerk.ts

```diff
@@ -132,15 +134,17 @@ export default class Clerk implements ClerkInterface {
     return this.#isReady;
   }
 
-  public constructor(key: string) {
+  public constructor(key: string, options?: Pick<ClerkInterface, 'proxyUrl'>) {
     key = (key || '').trim();
 
+    this.proxyUrl = options?.proxyUrl;
+
     if (isLegacyFrontendApiKey(key)) {
       if (!validateFrontendApi(key)) {
         errorThrower.throwInvalidFrontendApiError({ key });
       }
 
-      this.frontendApi = key;
+      this.frontendApi = options?.proxyUrl || key;
```

Comment:
🔧 I'd suggest to keeP it in a separate variable as both the `frontendApi` and the `proxyUrl` will be needed by ClerkJS. Also the `proxyUrl` needs to overwrite the frontendApi value in fapiClient.

Reply:
I see your point, I will refactor accordingly